src/styles/image.scss

```diff
@@ -0,0 +1,9 @@
+.images {
+  background-image: url(../images/imac.jpeg);
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), for SCSS files, it is recommended to use variables for colors, fonts, and other static values. Consider storing image paths as variables at the top of your SCSS files for easier maintenance and reusability.